docs/conf.py

```diff
@@ -32,8 +32,8 @@
 on_rtd = os.environ.get('READTHEDOCS') == 'True'
 on_netlify = os.environ.get('BUILD_VIDEOS') == 'True'
 on_travis = os.environ.get('TRAVIS') == 'True'
-#if not on_rtd and not on_netlify and not on_travis:
-#    exclude_patterns.append('videos')
+if not on_rtd and not on_netlify and not on_travis:
+    exclude_patterns.append('videos')
```

Comment:
You can just set one of these environment variables to test it as well.

`TRAVIS=True make html` for example.

Reply:
Of course. Neat - thanks!